docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md

```diff
@@ -0,0 +1,135 @@
+# Job manager and the JOB statements
+
+The long-term tasks running at the storage layer are called jobs. For example, there are jobs for `COMPACT`, `FLUSH`, and `STATS`. These jobs can be time-consuming if the data size in the graph space is large. The job manager helps you run, show, stop and recover the jobs.
+
+## SUBMIT JOB COMPACT
+
+The `SUBMIT JOB COMPACT` statement triggers the long-term RocksDB compact operation.
+
+```ngql
+nebula> SUBMIT JOB COMPACT;
++------------+
+| New Job Id |
++------------+
+| 40         |
++------------+
+```
+
+For more information about compact configuration, see Storage Service configuration (doc TODO).
+
+## SUBMIT JOB FLUSH
+
+The `SUBMIT JOB FLUSH` statement writes the RocksDB memfile in memory to the hard disk.
+
+## SUBMIT JOB STATS
+
+The `SUBMIT JOB STATS` statement starts a job that makes the statistics of the current graph space. Once this job succeeds, you can use the `SHOW STATS` statement to list the statistics. For more information, see [SHOW STATS](../7.general-query-statements/6.show/14.show-stats.md).
+
+## SHOW JOB <job_id>
+
+The Meta Service parses a `SUBMIT JOB` request into tasks and assigns them to the storaged processes. The `SHOW JOB <job_id>` statement shows the information about a specific job and all its tasks.
+
+The job ID is created when you run the `SUBMIT JOB` statement.
+
+```ngql
+nebula> SHOW JOB 96;
++----------------+---------------+------------+------------+------------+
+| Job Id(TaskId) | Command(Dest) | Status     | Start Time | Stop Time  |
++----------------+---------------+------------+------------+------------+
+| 96             | "FLUSH"       | "FINISHED" | 1606544069 | 1606544069 |
++----------------+---------------+------------+------------+------------+
+| 0              | "storaged2"   | "FINISHED" | 1606544069 | 1606544069 |
++----------------+---------------+------------+------------+------------+
+| 1              | "storaged0"   | "FINISHED" | 1606544069 | 1606544069 |
++----------------+---------------+------------+------------+------------+
+| 2              | "storaged1"   | "FINISHED" | 1606544069 | 1606544069 |
++----------------+---------------+------------+------------+------------+
+```
+
+The description of the return message is as follows.
+
+|Column|Description|
+|-|-|
+|Job Id(TaskId)|The first row shows the job ID, and the other rows show the task IDs.|
+|Command(Dest)|The first row shows the command executed, and the other rows show the storaged processes where the task is running.|
+|Status|Shows the status of the job or task. For more information about job status, see [Job status](#job-status).|
+|Start Time|Shows a timestamp indicating the time when the job or task enters the `RUNNING` phase.|
+|Stop Time|Shows a timestamp indicating the time when the job or task is `FINISHED`, `FAILED`, or `STOPPED`.|
```

Comment:
when the job or task gets `FINISHED`, `FAILED`, or `STOPPED`. OR when the job or task finishes, fails, or stops.